plugins/modules/proxmox.py

```diff
@@ -704,6 +723,7 @@ def main():
                                     unprivileged=ansible_to_proxmox_bool(module.params['unprivileged']),
                                     description=module.params['description'],
                                     hookscript=module.params['hookscript'],
+                                    timezone=module.params['timezone'],
```

Comment:
Do you know since when this option is supported in proxmoxer (the Python library used by this module)?

Reply:
Unfortunately not, and a look at [their changelog](https://github.com/proxmoxer/proxmoxer/blob/develop/CHANGELOG.md) did not help. 
The option doesn't even show up [searching in the source](https://github.com/search?q=repo%3Aproxmoxer%2Fproxmoxer%20timezone&type=code), so I'm assuming it's being handled alongside a bunch of other args? 
Had a really quick look though, so I can't tell for sure what is going on there. I have zero knowledge of the library so I can't really help much.

Reply:
proxmoxer library is just a wrapper around HTTP REST requests. You need to search it in the source code of Proxmoxer API.
https://github.com/proxmox/pve-container/commit/e6e308ae840abcaeb2bdc19fbaf6cc80b3efe034

Reply:
BTW, I haven't worked with PVE and ansible in a while. How create_instance would handle the fact that the timezone param is only valid for LXC and not for QEMU containers? Would it throw an error?

Reply:
That's a great point, thanks for the catch. I'm testing right now, will let you know asap.

Reply:
[https://pve.proxmox.com/wiki/Linux_Container](https://pve.proxmox.com/wiki/Linux_Container)
Proxmox does not use QEMU for containers, but only LXC.
The module itself only manages containers on Proxmox, not QEMU vms, so the problem should not arise.

Reply:
@felixfontein since you asked it. Do we need to do the check for version support or do we need to update the documentation for that option?

Reply:
@nxet I think I got it. It needs to be added in [minimum_version dict](https://github.com/ansible-collections/community.general/blob/1c59c8816760631154ee15e6e9a3321bb5e36e2f/plugins/modules/proxmox.py#L453). But Major version restriction is not enough, as for example, `timezone` was added in 6.3 and putting just `6` is a way to get an error on pre-6.3 instances. Could you please check if setting major.minor('timezone': 6.3) works?

Reply:
@nxet for some reason I can't make PR against your repository. Could you please accommodate my changes to your PR?
https://github.com/UnderGreen/community.general/pull/1/files

Reply:
@UnderGreen thanks a lot for your help! 
I would really like the commit to be authored by you, later today I'll fix the permissions on my fork.

Reply:
You should now be able to open PR on my fork, sorry for the delay

Reply:
NP, for sure. Glad to help the community after a massive break from contributing to open source.